### ISSUE TITLE
Bug: Cant connect to VPN Unlimited / Updated Server Certificates

### ISSUE THREAD

**Issue by sinjinsmythe** [qdm12/gluetun#2011]
### Is this urgent?

Yes

### Host OS

Synology

### CPU arch

x86_64

### VPN service provider

VPNUnlimited

### What are you using to run the container

docker run

### What is the version of Gluetun

Running version latest built on 2023-12-15T10:35:42.586Z (commit 4a6c229)

### What's the problem 🤔

No longer connects to VPN server, was working fine for a year, I suspect the updated servers have now affected the certificate I was using to connect, logs showing OpenSSL: error:0A000086:SSL routines::certificate verify failed.
I checked with VPNUnlimited and they said their engineers have completed the certificate updates to their servers.
Let me know what you need and I'll get it for you.

### Share your logs (at least 10 lines)

```plain text
2023-12-18T22:24:25.434155456Z Running version latest built on 2023-12-15T10:35:42.586Z (commit 4a6c229)
2023-12-18T22:24:25.434249708Z 
2023-12-18T22:24:25.434307333Z 🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
2023-12-18T22:24:25.434384872Z 🐛 Bug? https://github.com/qdm12/gluetun/issues/new
2023-12-18T22:24:25.434458260Z ✨ New feature? https://github.com/qdm12/gluetun/issues/new
2023-12-18T22:24:25.434530861Z ☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
2023-12-18T22:24:25.434604237Z 💻 Email? quentin.mcgaw@gmail.com
2023-12-18T22:24:25.434683413Z 💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-12-18T22:24:25.435354147Z 2023-12-18T22:24:25Z INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-12-18T22:24:25.435575737Z 2023-12-18T22:24:25Z INFO [routing] local ethernet link found: eth0
2023-12-18T22:24:25.435674101Z 2023-12-18T22:24:25Z INFO [routing] local ipnet found: 172.17.0.0/16
2023-12-18T22:24:25.450538719Z 2023-12-18T22:24:25Z INFO [firewall] enabling...
2023-12-18T22:24:25.479277405Z 2023-12-18T22:24:25Z INFO [firewall] enabled successfully
2023-12-18T22:24:27.029509864Z 2023-12-18T22:24:27Z INFO [storage] merging by most recent 17685 hardcoded servers and 17685 servers read from /gluetun/servers.json
2023-12-18T22:24:27.382919951Z 2023-12-18T22:24:27Z INFO Alpine version: 3.18.5
2023-12-18T22:24:27.388661676Z 2023-12-18T22:24:27Z INFO OpenVPN 2.5 version: 2.5.8
2023-12-18T22:24:27.395078622Z 2023-12-18T22:24:27Z INFO OpenVPN 2.6 version: 2.6.8
2023-12-18T22:24:27.398401990Z 2023-12-18T22:24:27Z INFO Unbound version: 1.17.1
2023-12-18T22:24:27.400378741Z 2023-12-18T22:24:27Z INFO IPtables version: v1.8.9
2023-12-18T22:24:27.400977973Z 2023-12-18T22:24:27Z INFO Settings summary:
2023-12-18T22:24:27.401185626Z ├── VPN settings:
2023-12-18T22:24:27.401264327Z |   ├── VPN provider settings:
2023-12-18T22:24:27.401332290Z |   |   ├── Name: vpn unlimited
2023-12-18T22:24:27.401400791Z |   |   └── Server selection settings:
2023-12-18T22:24:27.401473480Z |   |       ├── VPN type: openvpn
2023-12-18T22:24:27.401542018Z |   |       ├── Countries: romania
2023-12-18T22:24:27.401608006Z |   |       ├── Hostnames: ro.vpnunlimitedapp.com
2023-12-18T22:24:27.401680795Z |   |       └── OpenVPN server selection settings:
2023-12-18T22:24:27.401768796Z |   |           └── Protocol: UDP
2023-12-18T22:24:27.401842235Z |   └── OpenVPN settings:
2023-12-18T22:24:27.401908560Z |       ├── OpenVPN version: 2.5
2023-12-18T22:24:27.401973636Z |       ├── User: [set]
2023-12-18T22:24:27.402038800Z |       ├── Password: [set]
2023-12-18T22:24:27.402105163Z |       ├── Client crt: MII...w==
2023-12-18T22:24:27.402212514Z |       ├── Client key: MII...iU=
2023-12-18T22:24:27.402415792Z |       ├── Network interface: tun0
2023-12-18T22:24:27.402500481Z |       ├── Run OpenVPN as: root
2023-12-18T22:24:27.402597232Z |       └── Verbosity level: 1
2023-12-18T22:24:27.402669008Z ├── DNS settings:
2023-12-18T22:24:27.402765921Z |   ├── Keep existing nameserver(s): no
2023-12-18T22:24:27.402839722Z |   ├── DNS server address to use: 127.0.0.1
2023-12-18T22:24:27.402913461Z |   └── DNS over TLS settings:
2023-12-18T22:24:27.402982749Z |       ├── Enabled: yes
2023-12-18T22:24:27.403051563Z |       ├── Update period: every 24h0m0s
2023-12-18T22:24:27.403120876Z |       ├── Unbound settings:
2023-12-18T22:24:27.403184927Z |       |   ├── Authoritative servers:
2023-12-18T22:24:27.403251578Z |       |   |   └── cloudflare
2023-12-18T22:24:27.403315491Z |       |   ├── Caching: yes
2023-12-18T22:24:27.403381117Z |       |   ├── IPv6: no
2023-12-18T22:24:27.403446143Z |       |   ├── Verbosity level: 1
2023-12-18T22:24:27.403523319Z |       |   ├── Verbosity details level: 0
2023-12-18T22:24:27.403592145Z |       |   ├── Validation log level: 0
2023-12-18T22:24:27.403660358Z |       |   ├── System user: root
2023-12-18T22:24:27.403806585Z |       |   └── Allowed networks:
2023-12-18T22:24:27.403887249Z |       |       ├── 0.0.0.0/0
2023-12-18T22:24:27.403953749Z |       |       └── ::/0
2023-12-18T22:24:27.404014713Z |       └── DNS filtering settings:
2023-12-18T22:24:27.404082576Z |           ├── Block malicious: yes
2023-12-18T22:24:27.404153402Z |           ├── Block ads: no
2023-12-18T22:24:27.404219953Z |           ├── Block surveillance: no
2023-12-18T22:24:27.404286304Z |           └── Blocked IP networks:
2023-12-18T22:24:27.404353492Z |               ├── 127.0.0.1/8
2023-12-18T22:24:27.404423093Z |               ├── 10.0.0.0/8
2023-12-18T22:24:27.404486719Z |               ├── 172.16.0.0/12
2023-12-18T22:24:27.404552395Z |               ├── 192.168.0.0/16
2023-12-18T22:24:27.404618833Z |               ├── 169.254.0.0/16
2023-12-18T22:24:27.404695559Z |               ├── ::1/128
2023-12-18T22:24:27.404780535Z |               ├── fc00::/7
2023-12-18T22:24:27.404852074Z |               ├── fe80::/10
2023-12-18T22:24:27.404916137Z |               ├── ::ffff:127.0.0.1/104
2023-12-18T22:24:27.404992988Z |               ├── ::ffff:10.0.0.0/104
2023-12-18T22:24:27.405056914Z |               ├── ::ffff:169.254.0.0/112
2023-12-18T22:24:27.405123015Z |               ├── ::ffff:172.16.0.0/108
2023-12-18T22:24:27.405192578Z |               └── ::ffff:192.168.0.0/112
2023-12-18T22:24:27.405259079Z ├── Firewall settings:
2023-12-18T22:24:27.405325867Z |   └── Enabled: yes
2023-12-18T22:24:27.405389018Z ├── Log settings:
2023-12-18T22:24:27.405452356Z |   └── Log level: INFO
2023-12-18T22:24:27.405522945Z ├── Health settings:
2023-12-18T22:24:27.405593983Z |   ├── Server listening address: 127.0.0.1:9999
2023-12-18T22:24:27.405872012Z |   ├── Target address: cloudflare.com:443
2023-12-18T22:24:27.405947513Z |   ├── Duration to wait after success: 5s
2023-12-18T22:24:27.406038189Z |   ├── Read header timeout: 100ms
2023-12-18T22:24:27.406156278Z |   ├── Read timeout: 500ms
2023-12-18T22:24:27.406223491Z |   └── VPN wait durations:
2023-12-18T22:24:27.406294305Z |       ├── Initial duration: 6s
2023-12-18T22:24:27.406360393Z |       └── Additional duration: 5s
2023-12-18T22:24:27.406427856Z ├── Shadowsocks server settings:
2023-12-18T22:24:27.406494120Z |   └── Enabled: no
2023-12-18T22:24:27.406557533Z ├── HTTP proxy settings:
2023-12-18T22:24:27.406625284Z |   └── Enabled: no
2023-12-18T22:24:27.406689847Z ├── Control server settings:
2023-12-18T22:24:27.406777286Z |   ├── Listening address: :8000
2023-12-18T22:24:27.406843887Z |   └── Logging: yes
2023-12-18T22:24:27.406907725Z ├── OS Alpine settings:
2023-12-18T22:24:27.406975026Z |   ├── Process UID: 1024
2023-12-18T22:24:27.407042014Z |   ├── Process GID: 101
2023-12-18T22:24:27.407159353Z |   └── Timezone: europe/london
2023-12-18T22:24:27.407233679Z ├── Public IP settings:
2023-12-18T22:24:27.407306505Z |   ├── Fetching: every 12h0m0s
2023-12-18T22:24:27.407383194Z |   └── IP file path: /tmp/gluetun/ip
2023-12-18T22:24:27.407457557Z └── Version settings:
2023-12-18T22:24:27.407523408Z     └── Enabled: yes
2023-12-18T22:24:27.407588197Z 2023-12-18T22:24:27Z WARN OpenVPN 2.5 uses OpenSSL 3 which prohibits the usage of weak security in today's standards. vpn unlimited uses weak security which is out of Gluetun's control so the only workaround is to allow such weaknesses using the OpenVPN option tls-cipher "DEFAULT:@SECLEVEL=0". You might want to reach to your provider so they upgrade their certificates. Once this is done, you will have to let the Gluetun maintainers know by creating an issue, attaching the new certificate and we will update Gluetun.
2023-12-18T22:24:27.407783874Z 2023-12-18T22:24:27Z INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-12-18T22:24:27.407867188Z 2023-12-18T22:24:27Z INFO [routing] adding route for 0.0.0.0/0
2023-12-18T22:24:27.407937439Z 2023-12-18T22:24:27Z INFO [firewall] setting allowed subnets...
2023-12-18T22:24:27.408011415Z 2023-12-18T22:24:27Z INFO [routing] default route found: interface eth0, gateway 172.17.0.1, assigned IP 172.17.0.6 and family v4
2023-12-18T22:24:27.408085415Z 2023-12-18T22:24:27Z INFO TUN device is not available: open /dev/net/tun: no such file or directory; creating it...
2023-12-18T22:24:27.408167004Z 2023-12-18T22:24:27Z INFO [dns] using plaintext DNS at address 1.1.1.1
2023-12-18T22:24:27.408236692Z 2023-12-18T22:24:27Z INFO [http server] http server listening on [::]:8000
2023-12-18T22:24:27.408305581Z 2023-12-18T22:24:27Z INFO [healthcheck] listening on 127.0.0.1:9999
2023-12-18T22:24:27.408373382Z 2023-12-18T22:24:27Z INFO [firewall] allowing VPN connection...
2023-12-18T22:24:27.418651390Z 2023-12-18T22:24:27Z INFO [openvpn] --cipher is not set. Previous OpenVPN version defaulted to BF-CBC as fallback when cipher negotiation failed in this case. If you need this fallback please add '--data-ciphers-fallback BF-CBC' to your configuration and/or add BF-CBC to --data-ciphers.
2023-12-18T22:24:27.423118111Z 2023-12-18T22:24:27Z INFO [openvpn] OpenVPN 2.5.8 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Nov  2 2022
2023-12-18T22:24:27.423348251Z 2023-12-18T22:24:27Z INFO [openvpn] library versions: OpenSSL 3.1.4 24 Oct 2023, LZO 2.10
2023-12-18T22:24:27.429565569Z 2023-12-18T22:24:27Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]77.81.x.x:1194
2023-12-18T22:24:27.429803997Z 2023-12-18T22:24:27Z INFO [openvpn] UDP link local: (not bound)
2023-12-18T22:24:27.430044188Z 2023-12-18T22:24:27Z INFO [openvpn] UDP link remote: [AF_INET]77.81.x.x:1194
2023-12-18T22:24:33.405411012Z 2023-12-18T22:24:33Z INFO [healthcheck] program has been unhealthy for 6s: restarting VPN (see https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md)
2023-12-18T22:24:33.406094808Z 2023-12-18T22:24:33Z INFO [vpn] stopping
2023-12-18T22:24:33.406542552Z 2023-12-18T22:24:33Z INFO [vpn] starting
2023-12-18T22:24:33.434074022Z 2023-12-18T22:24:33Z INFO [firewall] allowing VPN connection...
2023-12-18T22:24:33.439432591Z 2023-12-18T22:24:33Z INFO [openvpn] --cipher is not set. Previous OpenVPN version defaulted to BF-CBC as fallback when cipher negotiation failed in this case. If you need this fallback please add '--data-ciphers-fallback BF-CBC' to your configuration and/or add BF-CBC to --data-ciphers.
2023-12-18T22:24:33.443306154Z 2023-12-18T22:24:33Z INFO [openvpn] OpenVPN 2.5.8 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Nov  2 2022
2023-12-18T22:24:33.443549357Z 2023-12-18T22:24:33Z INFO [openvpn] library versions: OpenSSL 3.1.4 24 Oct 2023, LZO 2.10
2023-12-18T22:24:33.449524685Z 2023-12-18T22:24:33Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]77.81.x.x:1194
2023-12-18T22:24:33.449679199Z 2023-12-18T22:24:33Z INFO [openvpn] UDP link local: (not bound)
2023-12-18T22:24:33.449775375Z 2023-12-18T22:24:33Z INFO [openvpn] UDP link remote: [AF_INET]77.81.x.x:1194
2023-12-18T22:24:33.568264314Z 2023-12-18T22:24:33Z INFO [openvpn] VERIFY ERROR: depth=2, error=self-signed certificate in certificate chain: C=US, ST=NY, L=New York, O=KeepSolid Inc., OU=KeepSolid Root CA, CN=KeepSolid Root CA, emailAddress=admin@keepsolid.com, serial=429164281094478856831696042475561970021707008630
2023-12-18T22:24:33.568526742Z 2023-12-18T22:24:33Z INFO [openvpn] OpenSSL: error:0A000086:SSL routines::certificate verify failed
2023-12-18T22:24:33.568610168Z 2023-12-18T22:24:33Z INFO [openvpn] TLS_ERROR: BIO read tls_read_plaintext error
2023-12-18T22:24:33.568714157Z 2023-12-18T22:24:33Z INFO [openvpn] TLS Error: TLS object -> incoming plaintext read error
2023-12-18T22:24:33.568824283Z 2023-12-18T22:24:33Z INFO [openvpn] TLS Error: TLS handshake failed
2023-12-18T22:24:33.569000460Z 2023-12-18T22:24:33Z INFO [openvpn] SIGTERM received, sending exit notification to peer
2023-12-18T22:24:33.569216826Z 2023-12-18T22:24:33Z INFO [openvpn] SIGTERM[soft,tls-error] received, process exiting
2023-12-18T22:24:33.571142076Z 2023-12-18T22:24:33Z INFO [vpn] retrying in 15s
```

### Share your configuration

```yml
docker run -it --rm --cap-add=NET_ADMIN -e VPN_SERVICE_PROVIDER="vpn unlimited" \
-v "/volume1/configs/gluetun:/gluetun" \
-e OPENVPN_USER=myemail@x.com -e OPENVPN_PASSWORD=mypasswordk \
-e TZ=europe/london -e PUID=1024 -e PGID=101 \
-e SERVER_COUNTRIES=Romania qmcgaw/gluetun
```

**Comment by qdm12** [comment]
Duplicate of #2005 and also, it should be fixed in the latest image (please comment on #2005 if yes/no). Thanks!
